core/domain/skill_services.py

```diff
@@ -514,3 +515,107 @@ def save_skill_summary(skill_summary):
     )
 
     skill_summary_model.put()
+
+
+def create_user_skill_mastery(user_id, skill_id, degree_of_mastery):
+    """Creates skill mastery of a user.
+
+    Args:
+        user_id: str. The user id of the logged in user.
```

Comment:
Here and in the docstrings below, do not imply that the user is currently logged in. We don't know that. It's fine to say something like "The user ID of the user." or "The user ID of the user for whom to create the model."

Ditto below (throughout this file).

Reply:
okay